integration-tests/api/__tests__/admin/payment.js

```diff
@@ -11,9 +13,47 @@ const adminHeaders = {
 
 let { simpleCustomerFactory, simplePaymentCollectionFactory } = {}
 
+const createV1PaymentSetup = async (dbConnection, payCol, api) => {
+  // create payment collection
+  payCol = await simplePaymentCollectionFactory(dbConnection, {
+    description: "paycol description",
+    amount: 1000,
+  })
+
+  // create payment session
+  const payColRes = await api.post(
+    `/store/payment-collections/${payCol.id}/sessions`,
+    {
+      provider_id: "test-pay",
+    }
+  )
+
+  // authorize payment session
+  await api.post(
+    `/store/payment-collections/${payCol.id}/sessions/batch/authorize`,
+    {
+      session_ids: payColRes.data.payment_collection.payment_sessions.map(
+        ({ id }) => id
+      ),
+    }
+  )
+
+  // get payment collection
+  const response = await api.get(
+    `/admin/payment-collections/${payCol.id}`,
+    adminHeaders
+  )
+
+  // return payment
+  return response.data.payment_collection.payments[0]
+}
+
 medusaIntegrationTestRunner({
-  env: { MEDUSA_FF_PRODUCT_CATEGORIES: true },
+  // env: { MEDUSA_FF_MEDUSA_V2: true },
```

Comment:
When I run the tests I just do `export MEDUSA_FF_MEDUSA_V2=true` so it's a bit easier to switch between the two in the terminal where I run the tests, but this works as well 😄

Reply:
I see, just thought this would be an easy switch to flip. I'll keep it as is for now